static/js/navigation/careers/Worker_populate-careers.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Should the filename be all lowercase?

Reply:
I wanted to emphasize that it a worker file. I haven't been able to find many conventions around naming workers in JS (I guess because it is a new technology). The one pattern I have seen is `populate-careers.worker.js`, but I also find trailing keywords often get missed.